components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/internal/model/AuthenticatedUserData.java

```diff
@@ -85,15 +86,15 @@ public static class Claim {
         private String uri;
 
         @JsonProperty("value")
-        private String value;
+        private Object value;
 
         public Claim() {
         }
 
         @JsonCreator
         public Claim(
                 @JsonProperty("uri") String uri,
-                @JsonProperty("value") String value) {
+                @JsonProperty("value") Object value) {
```

Comment:
Can we give different constructors here for string and a string list ?
We can keep the value as 'Object' type